contracts/interfaces/IIporOracle.sol

```diff
@@ -55,18 +54,20 @@ interface IIporOracle {
     /// @return accrued IBT price, represented in 18 decimals
     function calculateAccruedIbtPrice(address asset, uint256 calculateTimestamp) external view returns (uint256);
 
-    /// @notice Updates IPOR indexes for specified assets, accessible only by authorized updaters.
-    /// @dev Iterates through the 'indexesToUpdate' array, updating each asset's index and emitting {IporIndexUpdate} event.
-    ///      Special handling is applied for the '_stEth' asset. Function execution is restricted during pause state and
-    ///      requires the caller to be an authorized updater.
-    /// @param indexesToUpdate Array of 'UpdateIndexParams' containing asset address, new index value, update timestamp,
-    ///        and quasiIbtPrice (for '_stEth' asset).
-    /// @dev INPUT_ARRAYS_LENGTH_MISMATCH if 'indexesToUpdate' array is empty.
-    /// @dev ASSET_NOT_SUPPORTED if an asset in 'indexesToUpdate' is not supported.
-    /// @dev WRONG_INDEX_TIMESTAMP if the provided timestamp is either older than the last update timestamp or greater
-    ///         than the current block timestamp.
+    /// @notice Updates the Indexes based on the provided parameters.
+    /// @dev This function is external and can be called from other contracts.
+    /// The function accepts an array of the custom structure `UpdateIndexParams`.
+    /// The calldata specifies that the data should be stored in the calldata area which is read-only and cheaper to use than memory.
+    /// @param indexesToUpdate An array of `UpdateIndexParams` to be updated, in this function updateTimestamp and quasiIbtPrice are ignored.
     function updateIndexes(UpdateIndexParams[] calldata indexesToUpdate) external;
 
+    /// @notice Updates the Index and Quasi IBT price based on the provided parameters.
+    /// @dev This function is external and can be called from other contracts.
+    /// The function accepts an array of the custom struct `IIporOracle.UpdateIndexParams`.
```

Comment:
This is obvious so no need that in documentation. Code for this is self decriptive